src/authConfig.test.ts

```diff
@@ -0,0 +1,121 @@
+import { describe, it, expect, vi } from 'vitest';
+const { msalConfig, tokenConfig } = await import('./authConfig');
+import { LogLevel } from '@azure/msal-browser';
+
+const mockClientId = 'mock-client-id';
+const mockTenantId = 'mock-tenant-id';
+const mockWebAppUrl = 'http://example.com';
+
+describe('msalConfig', () => {
+  it('should have the correct auth', async () => {
```

Comment:
I like the test, but wouldn't it be better to name it 'should have the correct authority'? 
```suggestion
  it('should have the correct auth', async () => {
```

Reply:
I named it that because that was the name of property. But I'll rename it so test is "should have the correct auth property"